src/cascadia/TerminalControl/TermControlAutomationPeer.cpp

```diff
@@ -170,6 +177,23 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         };
     }
 
+    HRESULT TermControlAutomationPeer::GetHostUiaProvider(IRawElementProviderSimple** provider)
+    {
+        return S_OK;

```

Comment:
Is not populating provider, but returning "okay champ" valid here? We've seen weird behavior from UIA APIs when we return things-the-documentation-didnt-say-we-shouldnt-do (it's a whole dang thing)

Reply:
Should be okay, the previous code was already doing an "okay champ". This was just for abstracting since the WPF code doesn't